doc/tools/make_rst.py

```diff
@@ -1417,7 +1433,7 @@ def make_method_signature(
         # Use substitutions for abbreviations. This is used to display tooltips on hover.
         # See `make_footer()` for descriptions.
         for qualifier in method_def.qualifiers.split():
-            out += " |" + qualifier + "|"
+            out += " |" + sanitize_rst_literal(qualifier) + "|"
```

Comment:
Don't think we need to escape qualifiers, those are just text that we have predefined.